recipes/pivy/meta.yaml

```diff
@@ -0,0 +1,65 @@
+
+{% set name = "pivy" %}
+{% set version = "0.6.3" %}
+{% set commit = "6502b59ef12f193d5c36e48f07a3451ef7432a75" %}
+{% set sha256 = "d71e2275fa2b3d2659d0403e5b183c3433ebfa524d356696d682ac048a483058" %}
+
+package:
+    name: {{ name }}
+    version: {{ version }}
+
+source:
+    fn: {{ name }}.tar.gz
+    url: https://github.com/FreeCAD/pivy/archive/{{ commit }}.tar.gz
+    sha256: {{ sha256 }}
+
+
+build:
+    number: 0
+    skip: true  # [(win and not py36)]
+
+    features:
+        - vc14  # [(win and py36)]
+
+requirements:
+    build:
+        - toolchain
+        - nomkl  # [unix]
+        - vc 14  # [win]
+        - gcc  # [unix]
+        - msinttypes  # [win]
+        - cmake
+        - python
+        - swig  3.0.8
+        - coin3d
+        - colorama
+        - ninja
+        - qt 5.6*
+        # - soqt
+    run:
+        - vc 14  # [win]
+        - libgcc  # [unix]
+        - python
+        - coin3d
+        - qt 5.6.*
+        - pyside2
+        - libglu  # [linux]
+        # - soqt
+
+test:
+    imports:
+        - pivy.coin
+        # - pivy.gui.soqt
+
+about:
+    home: https://github.com/FreeCAD/pivy
+    license: BSD
```

Comment:
The license says it is a "ISC" license: https://github.com/FreeCAD/pivy/blob/master/LICENSE

Reply:
thanks, done.